test/functional/wiki_controller_test.rb

```diff
@@ -132,11 +132,23 @@ def teardown
     assert_equal flash[:notice], "Edits saved."
   end
 
+  test "basic user blocked from editing a locked wiki page" do
+    node(:organizers).add_tag('locked', rusers(:admin)) # lock the page with a tag
+    # then try editing it
+    assert_difference 'DrupalNodeRevision.count', 0 do
+      post :edit,
+          id: 'organizers'
+    end
+    assert_equal flash[:warning] , "This page is <a href='/wiki/power-tags#Locking'>locked</a>, and only <a href='/wiki/moderators'>moderators</a> can edit it."
+    assert_redirected_to node.path
```

Comment:
Ah - should be the last thing here -- this should be `node(:organizers).path` since we haven't made a variable called `node` in this test. Fingers crossed that this is the last thing!